docs/release-notes.md

```diff
@@ -10,6 +10,28 @@ toc_max_heading_level: 2
 
 Welcome to Semgrep release notes. This document provides an overview of the changes, additions, and fixes made in different versions.
 
+## April 2022
+
+### Version 0.87.0
+
+#### Additions
+
+- A new `focus-metavariable` operator that enables you to focus (or zoom in) the match on the code region delimited by a metavariable. This operator is useful for narrowing down the code matched by a rule, to focus on what matters. For more information, see [focus-metavariable documentation](/experiments/focus-metavariable/). ([#4453](https://github.com/returntocorp/semgrep/issues/4453))
+
+- Semgrep CI uses `GITHUB_SERVER_URL` to generate URLs if it is available.
```

Comment:
What is this used for?

Reply:
Good question. Please, @underyx, can you tell us more about this?

Reply:
CC @emjin

Reply:
It is for github enterprise users

Reply:
since their base url is not github.com but usually something like github.company.com or something

Reply:
This is a passive buff (improvement) for Enterprise then. :) (For Adam: There's no action the user needs to undertake, they are describing a change in behavior.) I wonder how might we convey enterprise vs community updates. A question for another time.

Reply:
Maybe we can just state that this is for GitHub Enterprise users then? @s-santillan 

Suggesting:

```
- For GitHub Enterprise users: Semgrep CI uses `GITHUB_SERVER_URL` to generate URLs if it is available.
```

Reply:
Yeah, let's do that 👍

Reply:
Thanks! Resolving.